packages/fx-core/resource/upgrade-change-logs.md

```diff
@@ -1,37 +1,22 @@
-
-### Environment Specific Configuration Files
-Teams Toolkit now supports creating multiple environments for a project, and you can customize the configurations for a specific environment, for example using different app names in the manifest for different environments. You can read more about what you can do in [this wiki](https://github.com/OfficeDev/TeamsFx/wiki/Enable-Preview-Features-in-Teams-Toolkit#managing-multiple-environments-in-teams-toolkit).
-
+After project upgrade, the new file structure will consist:
+* Project configurations for local debug setting, project setting etc.
+* Environment specific configurations for AAD, Manifest etc.
+* Project level template files including ARM and Manifest templates.
+ 
 ## Know what you need to do
-### Provision resource with bot
-If you have already provisioned the bot service before the migration, and you want to continue to use the bot service after the migration, please provision again. We will create a new bot service for this project, and other resources will not change. You can [read more](https://aka.ms/teamsfx-migration-guide#required-steps-after-migration) about why you need re-prevision.
+Below are the important things you should know about after upgrading your project.
 
-### Use exsiting bot
-If you want to reuse the existing bot that has been provision, please follow [this step](https://aka.ms/teamsfx-migration-guide#manual-work-to-use-existing-bot).
+* Execute [provision again if your project contains Bot / Messaging Extension](https://github.com/OfficeDev/TeamsFx/wiki/Upgrade-project-to-use-latest-Toolkit-features#reprovision-bot-project) capability.
+* Update properties if your project contains Azure [API Management (APIM) Service ](https://github.com/OfficeDev/TeamsFx/wiki/Upgrade-project-to-use-latest-Toolkit-features#customize-apim-service)
+* Connect to an [Azure SQL instance when local debug](https://github.com/OfficeDev/TeamsFx/wiki/Upgrade-project-to-use-latest-Toolkit-features#local-debug-with-sql) your Teams app.
+* Reset environment variables if you used an [existing AAD app](https://github.com/OfficeDev/TeamsFx/blob/dev/docs/fx-core/using-existing-aad.md#set-necessary-info-in-teamsfx-project).
+* [Change resource names](https://github.com/OfficeDev/TeamsFx/wiki/Upgrade-project-to-use-latest-Toolkit-features#change-resource-name-when-creating-a-new-environment) when creating a new environment.
```

Comment:
Looks there's no need to change the resource name when creating a new environment any more. @blackchoey can you please help double confirm it?

Reply:
Thanks for catching this, I updated the wiki to correct parameter file. This is needed because in the upgrade scenario, previously provisioned resources will be reused (except bot) so when creating a new environment we require to change resources names to avoid name conflict.